manifests/b/Benji377/Tooka/1.0.2/Benji377.Tooka.locale.en-US.yaml

```diff
@@ -18,9 +21,9 @@ Tags:
   - "utilities"
   - "organizer"
   - "tooka"
-Documentations: 

```

Comment:
Just a whitespace change lgtm.

Reply:
YAMLCreate by default removes any unnecessary trailing spaces in the YAML file, which is what's happening here. Are you expecting this to cause any problems?